libs/logic-apps-shared/src/designer-client-services/lib/base/template.ts

```diff
@@ -12,4 +13,6 @@ export class BaseTemplateService implements ITemplateService {
   }
 
   public openBladeAfterCreate = (workflowName: string): void => this.options.openBladeAfterCreate(workflowName);
```

Comment:
Can you please make sure we send workflowName as undefined when it is multi workflow.